nservicebus/azure/index.md

```diff
@@ -3,5 +3,31 @@ title: NServiceBus and Azure
 summary: Using Azure for endpoint hosting and to provide Transports and Persistence
 tags:
 - Azure
-reviewed: 2016-09-21
+reviewed: 2017-03-28
 ---
+
+NServiceBus helps create distributed .NET systems not only on premises but also in a public cloud environment. That is why it provides a variety of capabilities built for Microsoft Azure, the cloud closest to .NET. This covers hosting, Azure specific transports and persistence.
+
+
+## Hosting
```

Comment:
While this is the documented hosting on Azure when using hosts specially created for Azure CSes, I think it would make sense to list other options under the self-hosted model (App plan - webapp/jobs, SF, etc) Links could take to the samples, not just components.